test/support/factory.ex

```diff
@@ -113,4 +115,15 @@ defmodule Wanda.Factory do
       result: Map.get(attrs, :result, Enum.random([:passing, :warning, :critical]))
     }
   end
+
+  def result_factory(attrs) do
```

Comment:
`execution_result_factory` returns a `Result` while `result_factory` returns an `ExecutionResult`.
Could we maybe invert the name of those two?

Reply:
There you go dc1e66a59932e2b794e029dcfcba938e1e6d6be0

Reply:
LGTM :shipit: